### DISCUSSION TITLE
Does CPACS 3.3 version file supports with ACBuilder?

### DISCUSSION THREAD

**Discussion by kushaldgowda** [DLR-SL/CPACS#751] [Q&A] [answered]
Does CPACS 3.3 generated .xml file opens in ACBuilder in CEASIOM (MATLAB Version) ?

**Comment by MarAlder** [comment, marked as answer]
➡️  https://ceasiom.com/?page_id=169

**Reply by MarAlder** [reply]
I propose to contact the software vendors ⬆️. Would be nice if you keep us updated on answers.

**Reply by kushaldgowda** [reply]
Negative from that side. The format of .xml is different than ACBuilder .xml file.
Anyhow I ll try with this file with some modification. 
I "ll update the status later.

**Reply by MarAlder** [reply]
Nice. For some questions maybe @aidanjungo can help further. But I suggest to use the corresponding [CEASIOMpy GitHub page](https://github.com/cfsengineering/CEASIOMpy) for technical issues and questions.
